src/main/java/org/biouno/unochoice/util/SafeHtmlExtendedMarkupFormatter.java

```diff
@@ -0,0 +1,41 @@
+package org.biouno.unochoice.util;
+
+import com.google.common.base.Throwables;
+import hudson.markup.BasicPolicy;
+import hudson.markup.MarkupFormatter;
+import org.owasp.html.HtmlPolicyBuilder;
+import org.owasp.html.HtmlSanitizer;
+import org.owasp.html.HtmlStreamRenderer;
+import org.owasp.html.PolicyFactory;
+
+import java.io.IOException;
+import java.io.Writer;
+
+public class SafeHtmlExtendedMarkupFormatter extends MarkupFormatter {
+
+    public static SafeHtmlExtendedMarkupFormatter INSTANCE = new SafeHtmlExtendedMarkupFormatter();
+
+    private static final PolicyFactory POLICY = BasicPolicy.POLICY_DEFINITION.and(new HtmlPolicyBuilder()
+        .allowElements("input", "textarea")
+        .allowAttributes("id", "type", "name", "value", "placeholder", "disabled", "checked", "max", "maxlength", "min", "minlength", "multiple", "pattern", "readonly", "step").onElements("input")
+        .allowAttributes("id", "maxlength", "name", "placeholder", "disabled", "readonly", "wrap", "rows", "cols").onElements("textarea")
+        .toFactory());
```

Comment:
I tried to get all of the attributes that seemed important, but let me know if I missed any.